test/conformance/api/shared/util.go

```diff
@@ -61,7 +69,13 @@ func ValidateImageDigest(imageName string, imageDigest string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	if DigestResolutionExceptions.Has(ref.Context().RegistryStr()) {
```

Comment:
does this mean that after this change if I run conformance on my local minikube with side-loading I will see conformance pass when I've actually broken digest resolution, or is there some test somewhere that (even with side-loaded test images) will exercise the digest resolution path? (I guess with side-loading we already skip most of resolution, so maybe it's not a big deal if not, just want to be clear).

Reply:
Indeed, it might be better to refactor this to be able to call `t.Skip` in that scenario to at least point out that the verification got skipped.

Reply:
> it might be better to refactor this to be able to call t.Skip in that scenario to at least point out that the verification got skipped

We can do that, but calling `t.Skip` here would materially reduce coverage because this test has many facets.  I'll do that in a bit.

Reply:
Ehr, true as well. Maybe just a logline then?

Reply:
Oh sorry, what I'd meant was: let's refactor the test so that this check isn't here at all and write another more focused test for digest resolution that we can `t.Skip`.

Reply:
yeah, a separate test that explicitly tests resolution is what I was somewhat obliquely hoping for in the initial comment here. Sgtm.

Reply:
Oh I had a fun thought!  stay tuned :D

Reply:
Alright, check it out.  I plumbed through `t *testing.T` so that when we skip things we can add a token `t.Skip` with a note in those environments.  Changing things to use a proper `t.Run` would require a more extensive refactoring of this code, but I thought this might thread the needle.  I'd appreciate thoughts.